### DISCUSSION TITLE
Please download the latest image file to be able to update

### DISCUSSION THREAD

**Discussion by EarsofTin** [rern/rAudio#620] [General]
Attempted 20211114 update and got the message above.  Same result for 20211022 update but other updates such as 20211112 worked as usual.  Since I'm fairly adept at burning a new image it's no big deal but I'm wondering:
 
Is this a universal result for all users for this specific update?
Or is this the result of some setting or corruption unique to my installation?
And If I'm patient and stick with my current image will future updates become available?

Any comments appreciated.

rAudio 1 • false
5.10.74-1 • aarch64
20211004-1
Raspberry Pi 3 Model B Plus Rev 1.3
BCM2837B0 • 952MiB
4 @ 1.4GHz

**Comment by ceejayemm** [comment]
@EarsofTin 

Have a look at the file: 

/srv/http/data/addons/r1 

This is a text file editable with nano or similar . Mine currently contains the latest revision date (mine is 20211114) and nothing else.  I suspect yours in missing / empty (given the first line of your display above). If so create the file, or edit the existing, and set the date to (say) 20211113 and save it.  Then you should be able to reapply the latest update, after the update it should be the date of the latest revision.

If this is the case, this is an old RA-Re5/6 problem which still occurs occasionally in rAudio-1 for some reason.

Chris

**Reply by EarsofTin** [reply]
Perfect!  Thanks for the clever solution.  Note the updated version.

rAudio 1 • 20211114
5.10.74-1 • aarch64
20211004-1
Raspberry Pi 3 Model B Plus Rev 1.3
BCM2837B0 • 952MiB
4 @ 1.4GHz

**Reply by apastuszak** [reply]
> @EarsofTin
> 
> Have a look at the file:
> 
> /srv/http/data/addons/r1
> 
> This is a text file editable with nano or similar . Mine currently contains the latest revision date (mine is 20211114) and nothing else. I suspect yours in missing / empty (given the first line of your display above). If so create the file, or edit the existing, and set the date to (say) 20211113 and save it. Then you should be able to reapply the latest update, after the update it should be the date of the latest revision.
> 
> If this is the case, this is an old RA-Re5/6 problem which still occurs occasionally in rAudio-1 for some reason.
> 
> Chris

Thank you for this.  Fixed my problem.

**Reply by digixmax** [reply]
> @EarsofTin
> 
> Have a look at the file:
> 
> /srv/http/data/addons/r1
> 
> This is a text file editable with nano or similar . Mine currently contains the latest revision date (mine is 20211114) and nothing else. I suspect yours in missing / empty (given the first line of your display above). If so create the file, or edit the existing, and set the date to (say) 20211113 and save it. Then you should be able to reapply the latest update, after the update it should be the date of the latest revision.
> 
> If this is the case, this is an old RA-Re5/6 problem which still occurs occasionally in rAudio-1 for some reason.
> 
> Chris

The /srv/http/data/addons/r1 file on my RPi3 does have the date string 20210416 but the Add-On still shows "Please download the latest image file to be able to update" rather than the "Update" button.

Perhaps v20210416 is too old to be update-able to the current version?

**Reply by ceejayemm** [reply]
Then I think you need to do as it says and reflash the card with the latest image (i20220214) and then update to 20220225

Chris
